Response-Errors.md

```diff
@@ -0,0 +1,104 @@
+| `VALIDATION_ERROR` | `4211` | (**Input property**) should be true/false | You will get this error if you pass non-boolean value to CardHolderInformation.isCompany for example | Processing payment |
+| `VALIDATION_ERROR` | `4311` | WeiAmount must have non-zero value | You will get this error if your `fundTx.weiAmount` is zero | Creating payment |
+| `VALIDATION_ERROR` | `4411` | Vat number **{vat number here}** is invalid | You will get this error if you pass invalid VAT number in CardHolderInformation | Creating vendor / Processing Fiat payment | 
+| `VALIDATION_ERROR` | `4412` | Vat number is empty | You will get this error if you have vat number as property in CardHolderInformation, but it does not have a value | Creating vendor / Processing Fiat payment |
+
+### 5. Global Errors
+
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |--------------- | -------- | -------- |
+| `INTERNAL_SERVER_ERROR` | `5001` | Something went wrong. Contact your support for more information | General error | -
+| `BAD_REQUEST_ERROR` | `5111` | Bad request | You will get this error if you pass wrong input parameter | Every time bad input is passed |
+| `REQUESTS_RATE_LIMIT_ERROR` | `5211` | Too many requests, please try again later. | You will get this error if you exceed LimePay API rate limit | Every time rate limit is exceeded |
+
+
+### 6. Payment Errors
+
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |--------------- | -------- | -------- |
+| `PAYMENT_ERROR` | `6011` | Payment has already been processed  | You are allowed to create a payment and process it, but not re-process it | Processing payment |
+| `PAYMENT_ERROR` | `6012` | Payment can't be processed, because shopper is malicious |  You will get this error if a shopper is flagged as malicious one. You need to contact support once this error occurs | Processing payment |
+| `PAYMENT_ERROR` | `6111` | Payment can't be created, while another one is in processing state | You will get this error if you try to create new payment for a shopper, while another one is in processing state | Creating payment |
+| `PAYMENT_ERROR` | `6112` | Invalid payment amount | The Payment's amount must be bigger than 1 USD| Creating payment |
+| `PAYMENT_ERROR` | `6113` | Transactions with the given currency are not allowed | The provided currency is not supported | Creating payment |
+| `PAYMENT_ERROR` | `6201` | Can not get payment token | You will get this error if for some reason you can not get `Payment Token`. In this case contact support! | Retrieving payment token |
+| `PAYMENT_ERROR` | `6401` | **{payment transaction error}** | You will get this error if Bank returns to us that provided card information is invalid | Processing Fiat payment |
```

Comment:
You will get this error if Bank returns to us that provided card information is invalid
->
`You will get this error if LimePay's Payment Provider claims that the provided card information is invalid`